.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+pnpm exec commitlint --edit
```

Comment:
So instead of commitizen prompting us for a commit message and all that, this just ensures that we follow Conventional Commits (which Semantic Release requires). Wastes a little less time.

Reply:
nit: move this to `npx`? We moved away from pnpm